### PR TITLE
boskos: add a hypershift lease type

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

**Comment by ironcladlou** [comment]
cc @csrwng

File: core-services/prow/02_config/_boskos.yaml [hunk 1]
```diff
@@ -367,6 +367,10 @@ resources:
   - us-east1--gcp-quota-slice-69
   state: free
   type: gcp-quota-slice
+- max-count: 15
+  min-count: 15
+  state: free
+  type: hypershift-quota-slice
```
Comment (alvaroaleman) [inline core-services/prow/02_config/_boskos.yaml:373]: Please also add this to clusters/app.ci/prow/03_deployment/boskos_reaper.yaml

**Review by alvaroaleman — APPROVED** [review]

File: core-services/prow/02_config/_boskos.yaml (hunk 1 above)
Comment (ironcladlou) [reply core-services/prow/02_config/_boskos.yaml:373]: Was that supposed to happen during `make boskos-config`? It didn't. I updated `boskos_reaper.yaml` manually.

**Comment by alvaroaleman** [comment]
Nope, there is nothing that makes that happen today,

**Pull request merged**